### ISSUE TITLE
missing alt text prevents being used in a PDF/A-3a output

### ISSUE THREAD

**Issue by fraschfn** [Midbin/cades#15]
error: PDF/A-3a error: missing alt text
   ┌─ @preview/cades:0.3.1/lib.typ:21:9
   │
21 │   return image(bytes(result), width: width, height: height)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   │
   = hint: make sure your images and equations have alt text

**Comment by fraschfn** [comment]
there is already a pull request for this...
